### PR TITLE
update test suite output in installation guide 

### LINKED ISSUE #46498: BUG: pd.test() fails on Python v3.9.6
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [ ] I have confirmed this bug exists on the main branch of pandas.

### Reproducible Example

```python
>>> import pandas as pd
>>> pd.test()
```

### Issue Description

```
>>> import pandas as pd
>>> pd.test()
running: pytest --skip-slow --skip-network --skip-db /home/tms/.local/lib/python3.9/site-packages/pandas
======================================== test session starts =========================================
platform linux -- Python 3.9.6, pytest-7.1.1, pluggy-1.0.0
rootdir: /home/tms
plugins: hypothesis-6.39.4
collected 156958 items / 37 skipped                                                                  
# ...
= 3 failed, 146978 passed, 8670 skipped, 1318 xfailed, 3 xpassed, 169 warnings, 23 errors in 997.19s (0:16:37) =
```

Note the several failures, warnings and errors as well as the extended (997.19s) time.

### Expected Behavior

I'm a reasonably proficient Python developer attempting to evaluate whether `pandas` can solve a specific data imputation issue that has come up in a project. I'm therefore new to `pandas`, while not new to Python or the rest of my toolchain.

Surely a stable distribution of any package should pass its own documented tests! I see so many failures that I'm left doubting whether `pandas` is running at all. Since I am new to `pandas`, I have no way of knowing which if any of these failures, errors, and warnings are significant and which are not.

I have installed Pandas using `pip3` on a Rocky Linux v8.5 (CentOS 8) system running python 3.9. I did the installation using `pip3` and following the instructions in the [`pandas` documentation](https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html).

I expect any standard "stable" version to pass most of its published tests. In the [`pandas` documentation](https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html), the expected results are:

```
running: pytest --skip-slow --skip-network C:\Users\TP\Anaconda3\envs\py36\lib\site-packages\pandas
============================= test session starts =============================
platform win32 -- Python 3.6.2, pytest-3.6.0, py-1.4.34, pluggy-0.4.0
rootdir: C:\Users\TP\Documents\Python\pandasdev\pandas, inifile: setup.cfg
collected 12145 items / 3 skipped

..................................................................S......
........S................................................................
.........................................................................

==================== 12130 passed, 12 skipped in 368.339 seconds =====================
```

Significantly, I expect to have 0 failures, and 0 errors. I'm not sure about "xfailed", "xpassed", and "warnings".

I have tried using `pip3` to install various combinations of the 30+ dependencies mentioned in the documentation.  Seven of those cause `pip3` to fail, and so I skipped them.
```
SQLAlchemy
psycopg2
PyTables
zlib
PyQt4
xclip
xsel
```

By the time I installed all of the installable dependencies, the test results worsened:

```
= 4 failed, 148951 passed, 7431 skipped, 1330 xfailed, 3 xpassed, 184 warnings, 48 errors in 1081.06s (0:18:01) =
```

### Installed Versions

<details>
INSTALLED VERSIONS
------------------
commit           : 06d230151e6f18fdb8139d09abf539867a8cd481
python           : 3.9.6.final.0
python-bits      : 64
OS               : Linux
OS-release       : 4.18.0-348.12.2.el8_5.x86_64
Version          : #1 SMP Wed Jan 19 17:53:40 UTC 2022
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.4.1
numpy            : 1.22.3
pytz             : 2021.3
dateutil         : 2.8.2
pip              : 20.2.4
setuptools       : 50.3.2
Cython           : None
pytest           : 7.1.1
hypothesis       : 6.39.4
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : None
IPython          : None
pandas_datareader: None
bs4              : None
bottleneck       : 1.3.4
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : None
numba            : None
numexpr          : 2.8.1
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : 7.0.0
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.8.0
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None

</details>

### LINKED ISSUE #46504: DOC: outdated test suite output in installation guide
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#running-the-test-suite

### Documentation problem

The current output shown from `pd.test()` is outdated and was a source of confusion in #46498

### Suggested fix for documentation

This output should be updated so that the results are more consistent with the current size of the test suite (and perhaps a note should be added to indicate that these results are just an example of what you might see, it shouldn't exactly match)

### PR DESCRIPTION
**Pull request by eshirvana**
- [x] closes #46504
- [x] closes #46498
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: doc/source/getting_started/install.rst [hunk 1]
```diff
@@ -204,17 +204,28 @@ installed), make sure you have `pytest
 ::
 
     >>> pd.test()
-    running: pytest --skip-slow --skip-network C:\Users\TP\Anaconda3\envs\py36\lib\site-packages\pandas
-    ============================= test session starts =============================
-    platform win32 -- Python 3.6.2, pytest-3.6.0, py-1.4.34, pluggy-0.4.0
-    rootdir: C:\Users\TP\Documents\Python\pandasdev\pandas, inifile: setup.cfg
-    collected 12145 items / 3 skipped
+    running: pytest --skip-slow --skip-network --skip-db /home/user/anaconda3/lib/python3.9/site-packages/pandas
+    hi!!!
```
Comment (jreback) [inline doc/source/getting_started/install.rst:208]: can you remove the hi :->

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/getting_started/install.rst (hunk 1 above)
Comment (eshirvana) [reply doc/source/getting_started/install.rst:208]: @jreback sure, but it was what it's printed

File: doc/source/getting_started/install.rst [hunk 2]
```diff
@@ -204,17 +204,28 @@ installed), make sure you have `pytest
...
+
+    =================================== FAILURES ===================================
+
+    =============================== warnings summary ===============================
+
+    =========================== short test summary info ============================
+
+    = 1 failed, 146194 passed, 7402 skipped, 1367 xfailed, 5 xpassed, 197 warnings, 10 errors in 1090.16s (0:18:10) =
+
+This is just an example of what information is shown.You might see a slightly different result as what is shown above.
```
Comment (mroeschke) [inline doc/source/getting_started/install.rst:228]: Nit: Space between `shown. You`

Comment (eshirvana) [reply doc/source/getting_started/install.rst]: @mroeschke  fixed

File: doc/source/getting_started/install.rst [hunk 3]
```diff
@@ -228,7 +239,7 @@ Package                                                          Minimum support
 `python-dateutil <https://dateutil.readthedocs.io/en/stable/>`__ 2.8.1
 `pytz <https://pypi.org/project/pytz/>`__                        2020.1
 ================================================================ ==========================
-
+q
```
Comment (jreback) [inline doc/source/getting_started/install.rst:242]: can you revert this

**Review by mroeschke — APPROVED** [review]

**Review by jreback — APPROVED** [review]

**Pull request merged**